cli/src/commands/content/updateChannel.ts

```diff
@@ -118,11 +106,31 @@ export default class UpdateChannelCommand extends UploadCommandBase {
       assetIndices.avatarPhotoPath
     )
 
+    // Ensure actor is authorized to perform channel update
+    const requiredPermissions: ChannelActionPermission['type'][] = []
+    if (collaborators) {
+      requiredPermissions.push('ManageChannelCollaborators')
+    }
+    if (assetsToUpload) {
+      requiredPermissions.push('ManageNonVideoChannelAssets')
+    }
+    if (serializedMeta.length) {
+      requiredPermissions.push('UpdateChannelMetadata')
+    }
+    if (!(await this.hasRequiredChannelAgentPermissions(actor, channel, requiredPermissions))) {
+      this.error(
+        `Only channelOwner or collaborator with ${requiredPermissions} permission can update channel ${channelId}!`,
+        {
+          exit: ExitCodes.AccessDenied,
+        }
+      )
+    }
+
     const channelUpdateParameters = createType('PalletContentChannelUpdateParametersRecord', {
       expectedDataObjectStateBloatBond,
       assetsToUpload,
       assetsToRemove,
-      newMeta: metadataToBytes(ChannelMetadata, meta),
+      newMeta: serializedMeta,
```

Comment:
```suggestion
      newMeta: serializedMeta.length ? serializedMeta : null,
```
Runtime won't check `UpdateChannelMetadata` permissions only if provided metadata `isNone`
(the same fix is needed in `updateVideo.ts`)